spec/computer_player_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "spec_helper"
+require_relative "../lib/computer_player.rb"
+require_relative "../lib/grid.rb"
+
+RSpec.describe ComputerPlayer do
+  describe "#take_turn" do
+    it "gives the computer all choices that are available" do
```

Comment:
This test is a bit hard to follow. I think because the functionality is hard to follow too.